private-templates-service/server/package.json

```diff
@@ -0,0 +1,50 @@
+{
+	"name": "private-templates-service",
+	"version": "1.0.0",
+	"description": "Collaboration tool simplifying the workflow of developing adaptive cards.",
+	"main": "server.js",
+	"scripts": {
+		"test": "jest",
+		"test:watch": "jest --watch",
+		"start:prod": "npm install && node server.js",
+		"start": "tsc && node dist/server.js | bunyan",
+		"build": "tsc"
+	},
+	"author": "",
+	"license": "ISC",
+	"dependencies": {
+		"@testing-library/jest-dom": "^5.0.2",
+		"@testing-library/react": "^9.4.0",
+		"@types/async": "^3.0.5",
+		"@types/bluebird": "^3.5.29",
+		"@types/body-parser": "^1.17.1",
+		"@types/bunyan": "^1.8.6",
+		"@types/connect-mongo": "^3.0.0",
+		"@types/express-validator": "^3.0.0",
+		"@types/jws": "^3.2.1",
+		"@types/mongoose": "^5.5.43",
```

Comment:
dev-dependencies would be better I guess? Those are needed only before build

Reply:
I copy and pasted them into devDependencies.